### PR TITLE
cookiecutter 0.9 and fixes

### PR DESCRIPTION
**Pull request by youtux**
Several improvements over the formula: modernization, head url, declared resources and fix the python dependency.

### CODE REVIEW

File: Library/Formula/cookiecutter.rb [hunk 1]
```diff
@@ -12,14 +11,44 @@ class Cookiecutter < Formula
     sha1 "4c69f62dfffca5508feb0a5753dd0d4b7efb8ebe" => :mountain_lion
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
```
Comment (mikemcquaid) [inline Library/Formula/cookiecutter.rb]: Don't change this, it only needs this for Python 2.7 on versions earlier than Lion.

Comment (youtux) [inline Library/Formula/cookiecutter.rb]: Fixed. Still I'm wondering when to use the "if" and when not.

**Comment by youtux** [comment]
@BrewTestBot what's your problem?

**Comment by mikemcquaid** [comment]
@youtux Random Jenkins bug. Have queued a rebuild.

**Pull request closed without merging**